skipper.env

```diff
@@ -189,3 +189,6 @@ HOST_INSTALLER_ARGS
 ENABLE_SKIP_MCO_REBOOT
 ENABLE_SOFT_TIMEOUTS
 IMAGES_FLAVOR
+SERVICE_ACCOUNT_CLIENT_ID
```

Comment:
remove `OFFLINE_TOKEN` ? I see we also have `SECOND_OFFLINE_TOKEN` :thinking: maybe it's worth to remove it as well?